labcontrol/Controllers.py

```diff
@@ -36,16 +36,17 @@ def cmd_handler(self, cmd, params, queue, device_name): # this should recieve a
         else:
             print("No Parser Found. Will just pass params to command.")
 
-        # No get the command method. If there isn't a method, it should through an AttributeError.
-        method = getattr(self, cmd)
+        # Now get the command method. If there isn't a method, it should throw an AttributeError.
```

Comment:
Comment should be updated to match that it doesn't throw an error anymore.

Reply:
Updated! Will be staged in the next PR